DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -709,11 +709,6 @@ public class DcContext {
         get { return getConfigInt("show_emails") }
         set { setConfigInt("show_emails", newValue) }
     }
-
-    // do not use. use DcContext::isConfigured() instead
-    public var configured: Bool {
```

Comment:
in general, btw, i meanwhile see these functions that look like simple variables as harmful:

they look too cheap, the caller will not optimize the code around them and does not know that each call results in a database query, maybe even a complicated one.